pkgs/development/compilers/dmd/default.nix

```diff
@@ -1,25 +1,43 @@
-{ stdenv, lib, fetchFromGitHub
-, makeWrapper, unzip, which, writeTextFile
-, curl, tzdata, gdb, Foundation, git, callPackage
-, targetPackages, fetchpatch, bash
-, HOST_DMD? "${callPackage ./bootstrap.nix { }}/bin/dmd"
-, version? "2.097.2"
-, dmdSha256? "16ldkk32y7ln82n7g2ym5d1xf3vly3i31hf8600cpvimf6yhr6kb"
-, druntimeSha256? "1sayg6ia85jln8g28vb4m124c27lgbkd6xzg9gblss8ardb8dsp1"
-, phobosSha256? "0czg13h65b6qwhk9ibya21z3iv3fpk3rsjr3zbcrpc2spqjknfw5"
+{ stdenv
+, lib
+, fetchFromGitHub
+, makeWrapper
+, which
+, writeTextFile
+, curl
+, tzdata
+, gdb
+, Foundation
+, callPackage
+, targetPackages
+, fetchpatch
+, bash
+, installShellFiles
```

Comment:
I somewhat dislike using screen space this freely (also applies to later style changes in arrays). Is there an official guideline that says this is preferable?

Reply:
https://github.com/nix-community/nixpkgs-fmt

![Bildschirmfoto von 2022-08-26 22-53-12](https://user-images.githubusercontent.com/23431373/186989157-f4614ae3-5950-45de-a7b1-f71ce805c003.png)

Same applies to the `Inputs` lists. The [manual's style guide](https://nixos.org/manual/nixpkgs/stable/#sec-syntax) says that one-liners are also fine, but I find that it tends to look not great when you start adding `lib.optionals` to those, so you then switch to multi-line lists and now you have different styles of lists right next to each other which also doesn't look great. I think there's also an argument to be made about it being easier for Git to resolve merge conflicts with the multi-line variant? But I don't have to deal with such situations all too often so this is just from hearsay.

Reply:
Since there is an autoformatter tool, that to some extent means it's an official style recommendation. So the style change is warranted.

This is one of the reasons I hate autoformatters :).